nixos/modules/security/pam.nix

```diff
@@ -131,6 +131,18 @@ let
         '';
       };
 
+      duoSecurity = {
+        enable = mkOption {
+          default = false;
+          type = types.bool;
+          description = ''
+            If set, use the Duo Security pam module
+            <literal>pam_duo</literal> for authentication.  Requires
+            configuration of <literal>duo-unix</literal> package options.
```

Comment:
What does "Requires configuration of `duo-unix` package options." mean? There's no such option in NixOS.

Reply:
I've changed it to say security.duosec options -- is this better?

Reply:
Oh I see, in that case it should be `<option>security.duosec</option>`

Reply:
Ok, thanks!